buyer-api/src/utils/contracts.js

```diff
@@ -14,6 +14,7 @@ const dataExchange = DataExchangeContract.at(config.contracts.addresses.dataExch
 
 export {
   wibcoin,
+  wibcoin as token,
```

Comment:
Why is this needed?

Reply:
I know it's out of scope, but it felt bad to add more usage of the WIBToken as `wibcoin` knowing that we are changing it sooner or later. I'll revert this to preserve the scope of the PR.